src/server/process_request.c

```diff
@@ -645,6 +615,41 @@ process_request(int sfds)
 		}
 	}
 
+#if defined(PBS_SECURITY) && (PBS_SECURITY == KRB5)
+	if (conn->cn_credid != NULL &&
+	    conn->cn_auth_config != NULL &&
+	    conn->cn_auth_config->auth_method != NULL &&
+	    strcmp(conn->cn_auth_config->auth_method, AUTH_GSS_NAME) == 0) {
+
+		access_allowed = 0;
+
+		if (strcasecmp(server_host, request->rq_host)) {
+			/* always allow myself */
+			access_allowed = 1;
+		}
+
+		if (acl_check(get_sattr(SVR_ATR_acl_krb_realms), conn->cn_credid, ACL_Host)) {
```

Comment:
Before this check, please check whether its enabled "if (get_sattr_long(SVR_ATR_acl_krb_realm_enable))"

Reply:
Thx, done